utils.py

```diff
@@ -390,3 +391,40 @@ def get_gages(data, outlet_headnodes: list, dist: float) -> dict:
     del ss_gages, payload, fire_bounds_gages, fire_poly, coords, r
 
     return unique_gages
+
+
+def find_activate_gages(gages):
```

Comment:
Should it be called `find_active_gages` instead?

Reply:
I've been subconsciously typing activate instead on active all week! I blame conda.